### PR TITLE
Add documentation for wsl --mount

### PR DESCRIPTION
**Pull request by OneBlue**
This PR adds a new documentation page and links for `wsl --mount`

### CODE REVIEW

File: WSL/tutorials/wsl-mount-disk.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+description: Learn how to set up a disk mount in WSL 2 and how to access it.
+keywords: wsl, windows, windowssubsystem, gnu, linux, bash, disk, ext4, filesystem, mount
+ms.date: 06/08/2020
+ms.topic: article
+ms.localizationpriority: medium
+---
+
+# Get started mounting a linux disk in WSL 2 (preview)
+
+If you're dual booting to Linux, or simply want to access a linux disk format that isn't supported by Windows, WSL 2 can mount your disk and let you access its content.
```
Comment (craigloewen-msft) [inline WSL/tutorials/wsl-mount-disk.md:12]: Capitalize `Linux`

File: WSL/tutorials/wsl-mount-disk.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+wmic diskdrive list brief
+```
+
+The disks paths are available under the 'DeviceID' columns. Usually under the `\\.\PHYSICALDRIVE*` format.
+
+## List and select the partitions to mount in WSL 2
+
+Once the disk is identified, run:
+
+```powershell
```
Comment (craigloewen-msft) [inline WSL/tutorials/wsl-mount-disk.md:31]: This guide should mention that you need admin access.

File: WSL/wsl2-mount-disk.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
+---
+title: Mount disks in WSL2 (preview)
+description: Usage instructions to mount and attach physical disks in WSL2.
+keywords: BashOnWindows, bash, wsl, windows, windows subsystem for linux, windowssubsystem, windows 10, install, WSL2, version 2, disk, mount
+ms.date: 08/03/2020
+ms.topic: article
+ms.localizationpriority: high
+---
+
+# Mounting a disk using WSL 2 (preview)
```
Comment (craigloewen-msft) [inline WSL/wsl2-mount-disk.md:10]: This content is very similar to the tutorials content. Can we move this to tutorials for now and then put this content into the official docs when it is a full part of Windows?

File: WSL/wsl2-mount-disk.md [hunk 4]
```diff
@@ -9,9 +9,9 @@ ms.localizationpriority: medium
 
 # Get started mounting a linux disk in WSL 2 (preview)
 
-If you're dual booting to Linux, or simply want to access a linux disk format that isn't supported by Windows, WSL 2 can mount your disk and let you access its content.
+If you want to access a Linux disk format that isn't supported by Windows, you can use WSL 2 to mount your disk and access its content. No more need to dual-boot.
```
Comment (Biswa96) [inline WSL/wsl2-mount-disk.md:12]: "No more need to dual-boot." -- Now this is called monopoly.

File: WSL/wsl2-mount-disk.md [hunk 5]
```diff
@@ -155,20 +154,20 @@ This will make the block device available inside WSL 2 so it can be mounted manu
...
 ## Limitations
 
-- At this time, only whole disks can be attached to WSL 2 meaning that it's not possible to attach only a partition. Concretely this means that it's not possible to use `wsl --mount` to read a partition on the boot device, because that device can't be detached from Windows.
+- At this time, only entire disks can be attached to WSL 2, meaning that it's not possible to attach only a partition. Concretely, this means that it's not possible to use `wsl --mount` to read a partition on the boot device, because that device can't be detached from Windows.
 
 - USB flash drives are not supported at this time and will fail to attach to WSL 2. USB disks are supported though.
 
-- Only filesystems that are natively supported in the kernel can be mounted by `wsl --mount`. This means that it's not possible to use installed filesystem drivers (such as ntfs-3g for example) by calling `wsl --mount`.
\ No newline at end of file
+- Only filesystems that are natively supported in the kernel can be mounted by `wsl --mount`. This means that it's not possible to use installed filesystem drivers (such as ntfs-3g for example) by calling `wsl --mount`.
```
Comment (Biswa96) [inline WSL/wsl2-mount-disk.md:173]: Is the kernel here Linux kernel or ntoskrnl?

File: WSL/wsl2-mount-disk.md (hunk 4 above)
Comment (mattwojo) [reply WSL/wsl2-mount-disk.md:12]: ? How so? It's removing the restriction of having to boot into two different systems and saving a lot of time. It also doesn't eliminate anyone's ability to dual-boot. Not sure what the beef is here...?

Comment (Biswa96) [reply WSL/wsl2-mount-disk.md:12]: That line conveys the message that -- No one need to dual boot. It's not the 'ability', it's mindset.

Comment (mattwojo) [reply WSL/wsl2-mount-disk.md:12]: Seems a bit semantic to me. Do you have an alternative to suggest?

Comment (Biswa96) [reply WSL/wsl2-mount-disk.md:12]: In my opinion, just remove the last line.

Comment (mattwojo) [reply WSL/wsl2-mount-disk.md:12]: Hmm, not opposed to that. Thanks. I'll talk it over with the team.

File: WSL/wsl2-mount-disk.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+description: Learn how to set up a disk mount in WSL 2 and how to access it.
+keywords: wsl, windows, windowssubsystem, gnu, linux, bash, disk, ext4, filesystem, mount
+ms.date: 06/08/2020
+ms.topic: article
+ms.localizationpriority: medium
+---
+
+# Get started mounting a linux disk in WSL 2 (preview)
+
+If you want to access a Linux disk format that isn't supported by Windows, you can use WSL 2 to mount your disk and access its content. No more need to dual-boot.
```
Comment (craigloewen-msft) [inline WSL/wsl2-mount-disk.md:12]:
```suggestion
If you want to access a Linux disk format that isn't supported by Windows, you can use WSL 2 to mount your disk and access its content.
```

**Pull request merged**